src/blog/2018-04-03/use-git-to-select-files.pug

```diff
@@ -0,0 +1,67 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        
+        For a change log per package:
+
+        * [stryker changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker/CHANGELOG.md)
+        * [stryker-api changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-api/CHANGELOG.md)
+        * [stryker-mocha-runner changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-mocha-runner/CHANGELOG.md)
+        * [stryker-karma-runner changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-karma-runner/CHANGELOG.md)
+        * [stryker-typescript changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-typescript/CHANGELOG.md)
+        * [stryker-webpack-transpiler changelog](https://github.com/stryker-mutator/stryker/blob/master/packages/stryker-webpack-transpiler/CHANGELOG.md)
+
+        ## Sandboxes
+
+        Stryker only works on copies of your source code. A copy is called a *sandbox*. It creates as much of them as you want parallel test runners. 
+        You don't want a big sandbox. Just imagine what would happen if all your node_modules end up in there! That's why Stryker allows you to configure 
+        the files to load into it.
```

Comment:
There is a breakline here, I think this part should be on the previous line

Reply:
A breakline in markdown is nothing right. It will be part of the paragraph. Or did you mean something else?

Reply:
Ah right, this won't be rendered like that in the site